src/sections/ServicesHero/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import Section from "root/components/Section";
+import Typography from "root/components/Typography";
+import BackgroundVideoText from "root/components/BackgroundVideoText";
+
+import "./index.css";
+
+const column = 114;
+const gutter = 28;
+const eightColumns = 8 * column + 7 * gutter;
+const sixColumns = 6 * column + 5 * gutter;
+const breakpointMobile = 768;
+const breakpointDesktop = 1268;
+
+const renderCopy = () => (
+  <div>
+    <div styleName="left">
+      <Typography variant="h2" color="dark-blue" fontFamily="meta-serif">
+        Working together to achieve your goals
+      </Typography>
+    </div>
+    <div styleName="right">
+      <Typography color="dark-blue">
+        We’ll work alongside you from the early stages of product research and
+        design to development and deployment, all while continuously delivering
+        you product updates.
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDesktop = () => (
+  <Section>
+    <BackgroundVideoText
+      key="desktop"
+      labels={["A dynamic team to design", "and develop your product"]}
+      poster="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/oceanshot.jpg"
+      video="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/ocean-small.webm"
+      height="195px"
+      width={`${eightColumns}px`}
+    />
+    {renderCopy()}
+  </Section>
+);
+
+const renderMobile = () => (
+  <Section>
+    <div styleName="title">
+      <Typography variant="h1" color="dark-blue" fontFamily="meta-serif">
+        A dynamic team to design and develop your product
+      </Typography>
+      {renderCopy()}
+    </div>
+  </Section>
+);
+
+const renderTablet = () => (
+  <Section>
+    <BackgroundVideoText
+      key="tablet"
+      labels={["A dynamic team to", "design and develop", "your product"]}
+      poster="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/oceanshot.jpg"
+      video="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/ocean-small.webm"
+      height="330px"
+      width={`${sixColumns}px`}
+    />
+    {renderCopy()}
+  </Section>
+);
+
+const renderNoScript = () => (
+  <noscript>
+    <Section>
+      <div styleName="title">
+        <Typography variant="h1" color="dark-blue" fontFamily="meta-serif">
```

Comment:
Works perfectly, but you should change to a color similar to the video (i think lavender is good)

Reply:
I just use the default because I don't know how this is should really be, but i will change it to lavender

Reply:
Yes, you can merge after this